PackageInfo.g

```diff
@@ -58,7 +58,11 @@ ArchiveFormats := ".tar.gz",
 ##
 Status := "dev",
 
-AbstractHTML := "",
+AbstractHTML :=
+  "<span class=\"pkgname\">PackageManager</span> is a basic collection of \
+   simple functions for installing and removing \
+   <span class=\"pkgname\">GAP</span> packages, with the eventual aim of \
+   becoming a full pip-style package manager for the GAP system.",
```

Comment:
The second `GAP` should also be a `<span>` element.

Reply:
I've added a new commit to fix this.